src/system/config-wfrp4e.js

```diff
@@ -2103,22 +2104,22 @@ WFRP4E.PrepareSystemItems = function() {
                 scriptData: [
                     {
                         trigger: "dialog",
-                        label: "Penalty to all Tests (@effect.name)",
+                        label: game.i18n.format("WFRP4E.ConditionEffects.PenaltyToAllTests", {effect: game.i18n.localize("WFRP4E.ConditionName.Stunned")}),
```

Comment:
I feel like localizing script labels is a but much, most of them are hardly seen, and if it's necessary could just be replaced by the translation module?

Reply:
this would require replacement of whole object in the config, which i wanted to avoid in this case. And since there weren't so many - I wanted to keep it simple. but again, its your call - i can remove it.